app/app.js

```diff
@@ -111,6 +111,8 @@ angular.module('app',
         // otherwise will take care of routing the user to the specified url
         //$urlRouterProvider.otherwise('/views/map');
 
+    }).config(function ($urlMatcherFactoryProvider) {
+        $urlMatcherFactoryProvider.strictMode(false);
```

Comment:
Whats strictMode do?

Reply:
>Defines whether URLs should match trailing slashes, or not (the default behavior).

Since we want to allow not matching the trailing slash AFAIK, we need it off .